test/cql-pytest/test_restrictions.py

```diff
@@ -84,7 +84,6 @@ def test_multiple_restrictions_on_ck(cql, table2, scylla_only):
 # number of values on the right-hand-side of the restriction. Scylla should
 # cleanly report the error - and not silently ignore it or even crash as in
 # issue #13241.
-@pytest.mark.skip("Crashes due to issue #13241")
```

Comment:
Thanks for un-skipping this test, but there is an additional test you need to unskip (and check that they pass) - in test/cql-pytest/cassandra_tests/validation/operations/select_multi_column_relation_test.py 
In general, trying "git grep 13241" is a good way to find the pre-existing tests that refer to some issue you're solving.

Reply:
Note that that second test won't begin to pass (because #4244 is still open) but it will at least stop crashing and won't need to be skipped.

Reply:
This comment was also not addressed.